### PR TITLE
Add a cookies.txt parser

### PR DESCRIPTION
**Pull request by cmdcolin**
Addresses #5 

This is a very simple text parsing that depends on the file extension, so it uses the cookies.txt parser if the file format is not ".json"

Tested downloading some releases and worked ok. Let me know if there are any suggestions :)

### CODE REVIEW

**Comment by Ezwen** [comment]
Thank you very much, looks good! 

I will test it as soon as I can, and make comments if needed. I hope before the end of next week.

File: README.adoc [hunk 1]
```diff
@@ -9,14 +9,17 @@ $ java -jar ./bandcamp-collection-downloader.jar -d myMusicFolder myBandcampAcco
...
-                            A JSON file with valid bandcamp credential cookies.
-                            "Cookie Quick Manager" can be used to obtain this file after logging into bandcamp.
-                            (visit https://addons.mozilla.org/en-US/firefox/addon/cookie-quick-manager/).
-                            If no cookies file is provided, cookies from the local Firefox installation are used (Windows and Linux
+                            A JSON or txt file with valid bandcamp credential cookies.
+                            "Cookie Quick Manager" can be used to obtain a JSON file after logging into bandcamp.
+                            (visit https://addons.mozilla.org/en-US/firefox/addon/cookie-quick-manager/)
+                            "or the "cookies.txt" extension"
+                            (https://chrome.google.com/webstore/detail/cookiestxt/njabckikapfpffapmjgojcnbfjonfjfg?hl=en)
+                            can be used to obtain a cookies.txt file on Chrome.
```
Comment (Ezwen) [inline README.adoc:21]: Here I would rephrase like this:

A file containing valid bandcamp credential cookies.
The file must either be obtained using the Firefox extension "Cookie Quick Manager" (https://addons.mozilla.org/en-US/firefox/addon/cookie-quick-manager/) or using the Chrome extension "cookies.txt" (https://chrome.google.com/webstore/detail/cookiestxt/njabckikapfpffapmjgojcnbfjonfjfg).
If no cookies file is provided, cookies from the local Firefox installation are used (Windows and Linux).

WDYT?

File: src/main/kotlin/bandcampcollectiondownloader/Args.kt [hunk 2]
```diff
@@ -10,7 +10,13 @@ data class Args(
...
-                description = ["A JSON file with valid bandcamp credential cookies.", """"Cookie Quick Manager" can be used to obtain this file after logging into bandcamp.""", "(visit https://addons.mozilla.org/en-US/firefox/addon/cookie-quick-manager/).", "If no cookies file is provided, cookies from the local Firefox installation are used (Windows and Linux only)."])
+                description = ["A JSON or txt file with valid bandcamp credential cookies.",
+                            """"Cookie Quick Manager" can be used to obtain a JSON file after logging into bandcamp.""",
+                            "(visit https://addons.mozilla.org/en-US/firefox/addon/cookie-quick-manager/)",
+                            """"or the "cookies.txt" extension"""",
+                            "(https://chrome.google.com/webstore/detail/cookiestxt/njabckikapfpffapmjgojcnbfjonfjfg?hl=en)",
+                            "can be used to obtain a cookies.txt file on Chrome.",
+                            "If no cookies file is provided, cookies from the local Firefox installation are used (Windows and Linux) only)."])
```
Comment (Ezwen) [inline src/main/kotlin/bandcampcollectiondownloader/Args.kt:19]: See my comment in the README.

File: src/main/kotlin/bandcampcollectiondownloader/Args.kt [hunk 3]
```diff
@@ -39,4 +45,4 @@ data class Args(
         @CommandLine.Option(names = ["-v", "--version"], versionHelp = true, description = ["Display the version and exits."])
         var version: Boolean = false
 
-)
\ No newline at end of file
+)
```
Comment (Ezwen) [inline src/main/kotlin/bandcampcollectiondownloader/Args.kt]: I think you can remove this poor lost new line :)

File: src/main/kotlin/bandcampcollectiondownloader/CookiesManagement.kt [hunk 4]
```diff
@@ -125,4 +142,4 @@ object CookiesManagement {
         return Cookies(cookiesFilePath, result)
     }
 
-}
\ No newline at end of file
+}
```
Comment (Ezwen) [inline src/main/kotlin/bandcampcollectiondownloader/CookiesManagement.kt]: Same here

**Review by Ezwen — CHANGES_REQUESTED** [review]
Looks good! If you have time to address my very few comments that would be great.

Last request: could you if possible drop a line in the README under the section "Bandcamp authentication", to mention the possibility to use the Chrome extension?

thanks again for the contribution!

File: README.adoc (hunk 1 above)
Comment (cmdcolin) [reply README.adoc]: I think that is fine, I replaced it with this text

**Comment by cmdcolin** [comment]
Added some of those changes, I think that is fine. No unit tests for cookies.txt but I "tested it" by using it to download some albums and it worked well :)

**Comment by Ezwen** [comment]
Thank you :+1: ! Only this change is missing (the only one I did not put inside the diff view) and we are good to merge:

> Last request: could you if possible drop a line in the README under the section "Bandcamp authentication", to mention the possibility to use the Chrome extension?

**Comment by cmdcolin** [comment]
Added a short note about this in the authentication section now
